demo/examples/04-jobs.js

```diff
@@ -0,0 +1,36 @@
+import { getJobsStatus, queuePrint } from 'inkmap';
+import { tap } from 'rxjs/operators';
+import { downloadBlob, getJobStatus } from '../../src/main';
+
+const root = document.getElementById('example-04');
+const btn = /** @type {CustomButton} */ root.querySelector('custom-button');
+const bars = /** @type {CustomProgresses} */ root.querySelector(
+  'custom-progresses'
+);
+const spec = /** @type {PrintSpec} */ root.querySelector('print-spec');
+
+// subscribe to the long-running observable
+// beware of managing the unsubscription
+getJobsStatus()
+  .pipe(
+    tap((jobs) => {
```

Comment:
Don't know if it wouldn't be clearer to have it in the `subscribe` method ?

Reply:
Yes, I took the habit to always do the maximum in the pipe operator, but here it might be too much.